### DISCUSSION TITLE
Help needed direct binding (ROBB ROB_200-011-0) dimmer to group

### DISCUSSION THREAD

**Discussion by TimGW** [Koenkk/zigbee2mqtt#25076] [Devices]
Hi, I would like to directly bind/control my zigbee group via [this dimmer device ](https://www.zigbee2mqtt.io/devices/ROB_200-011-0.html) which I think should support direct binding according to the [documentation](https://www.robbshop.nl/media/4b/73/8b/1729854646/ROB_200-011-0_download_2.pdf). However I am not sure what I'm doing wrong. Any help would be very much appreciated.

This is how the settings look of the dimmer. 

![Screenshot from 2024-12-05 17-19-37](https://github.com/user-attachments/assets/0061ba91-d616-4bbe-8cac-8b06da85a955)

Binding seems to be succesfull, but no matter what I try, pressing the dimmer switch results in only controlling the light itself, not the group. I tried the same steps with an old Hue dimmer switch, that seems to be working like expected.

**Comment by TimGW** [comment]
I guess the device could not do it after all :(

**Comment by vdhmax** [comment]
Not sure if this helps, but I too have issues with ROBB devices, in my case a switch.

I have ROBB Switches (for instance: ROBB ROB_200-007-0). I can bind the switch to a light, but the dimming feature doesn't seem to work. It works fine if I set up an automation in Home Assistant to do the dimming, but:
- it's not smooth for lights that don't support transitions. 
- it's buggy when using multiple lights and/or light groups

Hence I'd like to use the dimming directly by binding the switch to a light from Z2M to see if it works better / more smoothly. However I can only get the light to turn on and off in this way, dimming does nothing.
